faroese/COPYING

```diff
@@ -0,0 +1,12 @@
+﻿Copyright © 2011 Fróðskaparsetur Føroya (University of Faroe Islands)
+
+These files are published under the following open source licenses:
+
+GNU GPL version 2.0
+GNU LGPL version 2.1
+Mozilla MPL version 1.1

```

Comment:
To be clear, are these "or any later version" or restricted to the exact versions specified?

Also, is there an home page or similar for this word list?  If so that would be useful to note here.

Reply:
As far as I know the word list was released with the above licenses by The University of the Faroe Islands and then forgotten about. I believe it was/is used in Hunspell. The list (along with the license) was to be found at stava.fo which is unresponsive now. I found it via web.archive.org